metadata/src/test/java/org/apache/kafka/metadata/bootstrap/BootstrapDirectoryTest.java

```diff
@@ -81,18 +81,17 @@ public void testReadFromEmptyConfiguration() throws Exception {
     @Test
     public void testReadFromConfigurationWithAncientVersion() throws Exception {
         try (BootstrapTestDirectory testDirectory = new BootstrapTestDirectory().createDirectory()) {
-            assertEquals(BootstrapMetadata.fromVersion(MetadataVersion.MINIMUM_BOOTSTRAP_VERSION,
-                    "the minimum version bootstrap with metadata.version 3.3-IV0"),
+            assertThrows(IllegalArgumentException.class, () ->
                 new BootstrapDirectory(testDirectory.path(), Optional.of("3.0")).read());
         }
     }
 
     @Test
     public void testReadFromConfiguration() throws Exception {
         try (BootstrapTestDirectory testDirectory = new BootstrapTestDirectory().createDirectory()) {
-            assertEquals(BootstrapMetadata.fromVersion(MetadataVersion.IBP_3_3_IV2,
-                    "the configured bootstrap with metadata.version 3.3-IV2"),
-                new BootstrapDirectory(testDirectory.path(), Optional.of("3.3-IV2")).read());
+            assertEquals(BootstrapMetadata.fromVersion(MetadataVersion.IBP_3_3_IV3,
+                    "the configured bootstrap with metadata.version 3.3-IV3"),
+                new BootstrapDirectory(testDirectory.path(), Optional.of("3.3-IV3")).read());
```

Comment:
Should we change `3.0-IV0` in `testReadFromConfigurationFile` below?

Reply:
I think the code you're referencing was removed in a subsequent commit.